### ISSUE TITLE
How to define custom nodejs source code dir

### ISSUE THREAD

**Issue by mattbrun** [vercel/pkg#326]
Hello everyone,
in the case I need to build from nodejs sources instead of downloading the precompiled ones, is it possibile to specify the nodejs sources direcotry path instead of having pkg download it from git?

My situation is the following:
- I correctly created a binary with `pkg` under a `x86` architecture, but i need to create the binary for the `armv7` architecture and `pkg` says i cannot crosscompile on my system (I guess there's not this feature yet...);
- So i tried to run `pkg` directly on my `armv7` system (a Raspberry), but since there's no prebuilt binary for the latest versions of node and pkg, it tries to get the nodejs source code from git;
- The nodejs source code download fails because `pkg` tries to download it under `/tmp`, but there's not enough space on `/tmp` because it's a RAMfs with limited space - around 470MB when I need at least 642MB.

In conclusion I see the following solutions:
- downgrade `pkg` installed version and nodejs version - I need node9, but i may try to go with node8 if it resolves the issue;
- use `pkg` with the `--build` option if there is the possibility to give the nodejs sources directory or to say where to download the nodejs git sources - but I cannot find such possibilities...

Any other ideas?

Thanks a lot for your help and hard work!
M.

**Comment by biancode** [comment]
a PKG_TMP_PATH variable would be great here or an option to set tmp by --tmp
